vie.ini

```
@@ -3,10 +3,442 @@
+commands.banip.success.players=Banned IP address {%0} belonging to {%1}
+commands.banip.success=Đã cấm địa chỉ IP {%0}
+commands.banip.usage=/ban-ip <địa chỉ IP|tên người chơi> [lý do ...]
+commands.banlist.ips=Có tất cả {%0} địa chỉ IP đã bị cấm:
+commands.banlist.players=There are {%0} total banned players:
+commands.banlist.usage=/banlist [ips|players]
+commands.clear.failure.no.items=Could not clear inventory of {%0}, no items to remove
+commands.clear.success=Cleared the inventory of {%0}, removing {%1} items
+commands.clear.testing={%0} has {%1} items that match the criteria
+commands.defaultgamemode.success=The world's default game mode is now {%0}
+commands.defaultgamemode.usage=/defaultgamemode <mode>
+commands.deop.message=You have been de-opped
+commands.deop.success=De-opped {%0}
+commands.deop.usage=/deop <player>
+commands.difficulty.success=Set game difficulty to {%0}
+commands.difficulty.usage=/difficulty <new difficulty>
+commands.effect.failure.notActive.all=Couldn't take any effects from {%0} as they do not have any
+commands.effect.failure.notActive=Couldn't take {%0} from {%1} as they do not have the effect
+commands.effect.notFound=There is no such mob effect with ID {%0}
+commands.effect.success.removed.all=Took all effects from {%0}
+commands.effect.success.removed=Took {%0} from {%1}
+commands.effect.success=Given {%0} * {%1} to {%2} for {%3} seconds
+commands.effect.usage=/effect <player> <effect> [seconds] [amplifier] [hideParticles] OR /effect <player> clear
+commands.enchant.noItem=The target does not hold an item
+commands.enchant.notFound=There is no such enchantment with ID {%0}
+commands.enchant.success=Enchanting succeeded for {%0}
+commands.enchant.usage=/enchant <player> <enchantment ID> [level]
+commands.gamemode.success.other=Set {%1}'s game mode to {%0}
+commands.gamemode.success.self=Set own game mode to {%0}
+commands.gamemode.usage=/gamemode <mode> [player]
+commands.generic.notFound=Unknown command. Try /help for a list of commands
+commands.generic.num.tooBig=The number you have entered ({%0}) is too big, it must be at most {%1}
+commands.generic.num.tooSmall=The number you have entered ({%0}) is too small, it must be at least {%1}
+commands.generic.permission=You do not have permission to use this command
+commands.generic.player.notFound=That player cannot be found
+commands.generic.usage=Usage: {%0}
+commands.give.item.notFound=There is no such item with name {%0}
+commands.give.success=Given {%0} * {%1} to {%2}
+commands.give.tagError=Data tag parsing failed: {%0}
+commands.help.header=--- Showing help page {%0} of {%1} (/help <page>) ---
+commands.help.usage=/help [page|command name]
+commands.kick.success.reason=Kicked {%0} from the game: '{%1}'
+commands.kick.success=Kicked {%0} from the game
+commands.kick.usage=/kick <player> [reason ...]
+commands.kill.successful=Killed {%0}
+commands.me.usage=/me <action ...>
```

```suggestion
commands.me.usage=/me <hoạt động ...>
```
`action` : `hoạt động` or ?